sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -95,8 +96,7 @@ namespace Azure { namespace Core {
      * @param fractionFormat The format for the fraction part of the Datetime. Only supported by
      * RFC 3339.
      *
-     * @throw std::length_error If year exceeds 9999.
-     * @throw std::invalid_argument If \p format is not recognized.
+     * @throw std::length_error If year exceeds 9999, or if \p format is not recognized.
```

Comment:
wrong exception type :(

should be `invalid_argument`

Reply:
Oops, the comment is wrong.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/pull/950